addons/project_account/tests/test_project_profitability.py

```diff
@@ -47,3 +47,46 @@ def test_project_profitability(self):
             },
             'The profitability data of the project should return the total amount for the revenues and costs from tha AAL of the account of the project.'
         )
+
+    def test_project_profitability_with_manufacturing_orders(self):
+        #  When the analytic account has some manufacturing orders , they should be counted only once
+        mrp_account = self.env['ir.module.module']._get('mrp_account')
+        if mrp_account.state != 'installed':
+            self.skipTest("mrp_account module is not installed")
+        product = self.env['product.product'].create({
+            'name': 'Product',
+            'type': 'product',
+            'standard_price': 233.0,
+        })
+        component = self.env['product.product'].create({
+            'name': 'Component',
+            'type': 'product',
+            'standard_price': 10.0,
+        })
+        bom = self.env['mrp.bom'].create({
+            'product_id': product.id,
+            'product_tmpl_id': product.product_tmpl_id.id,
+            'product_qty': 1.0,
+            'type': 'normal',
+            'bom_line_ids': [
+                (0, 0, {'product_id': component.id, 'product_qty': 1.0}),
```

Comment:
You could use `Command` class. 🙂 

to import it: `from odoo import Command`
```suggestion
                Command.create({'product_id': component.id, 'product_qty': 1.0}),
```